osquery/tables/system/windows/drivers.cpp

```diff
@@ -110,37 +119,37 @@ Status getDeviceProperty(const device_infoset_t& infoset,
                   "Error getting buffer size for device property");
   }
 
-  std::unique_ptr<BYTE, decltype(freePtr)> drvBuff(
-      static_cast<PBYTE>(malloc(buffSize)), freePtr);
-  if (drvBuff == nullptr) {
+  std::unique_ptr<BYTE, decltype(kFreePtr)> drv_buff(
+      static_cast<PBYTE>(malloc(buff_size)), kFreePtr);
```

Comment:
You could use `unique_ptr` version for raw arrays, it would be a bit simpler, I think. If you like so :)
```c++
auto drv_buff = std::make_unique<BYTE[]>(buff_size);
```